src/components/Select.tsx

```diff
@@ -157,7 +158,9 @@ export default class Select<OptionType> extends React.PureComponent<
           testID={`${testIdPrefix}-label`}
           onPress={
             disabled
-              ? undefined
+              ? showToast
+                ? () => showToast()
+                : undefined
```

Comment:
Select is a general ui component. dont add props like `showToast` that cater to specific use case in app